code/modules/spells/roguetown/archfey_warlock.dm

```diff
@@ -0,0 +1,48 @@
+/obj/effect/proc_holder/spell/invoked/archfey_warlock_strip
+	name = "Strip Clothes"
+	overlay_state = "bcry"
+	releasedrain = 80
+	charge_max = 3 MINUTES
+	range = 1
+	cast_without_targets = TRUE
+	sound = 'sound/magic/churn.ogg'
+	invocation_type = "none" //can be none, whisper, emote and shout
+
+/obj/effect/proc_holder/spell/invoked/strip/cast(list/targets, mob/user)
+	. = ..()
+	user.emote("giggle")
+	var/mob/living/target = targets[1]
+	if(iscarbon(target))
+		var/obj/item/object = target.get_item_by_slot(pick(SLOT_GLOVES,SLOT_SHOES,SLOT_HEAD))
+		if(!istype(object, /obj/item/clothing/head/roguetown/helmet))//Can't take helmets!
+			target.dropItemToGround(object)
+		user.log_message("has stripped a piece of clothing from [key_name(target)] via spell", LOG_ATTACK)
+		target.log_message("has had a piece of clothing stripped by [key_name(user)] via spell", LOG_ATTACK)
+	return TRUE
+
+/obj/effect/proc_holder/spell/targeted/archfey_warlock_seelie_kiss
+	name = "Regenerative Kiss"
+	overlay_state = "heal"
+	releasedrain = 0
+	charge_max = 2 MINUTES
+	range = 1
+	invocation_type = "none" //can be none, whisper, emote and shout
+
+/obj/effect/proc_holder/spell/targeted/seelie_kiss/cast(list/targets, mob/user)
+	. = ..()
+	if(iscarbon(targets[1]))
+		var/mob/living/carbon/target = targets[1]
+		var/obj/item/reagent_containers/powder/K = new /obj/item/reagent_containers/powder/AFSK(target.loc)
```

Comment:
Who made this shit why would anyone make a powder just to transfer its reagents what wtf

Reply:
it was copied over the default seelie kiss

the amount of healing should be only 4oz? but i am also not sure

Reply:
alright we just tested the kiss as it is in game right now and it healed a grand total of 2 damage

it might need to buff it more honestly 

![image](https://github.com/user-attachments/assets/8ecd663f-b718-46b4-8e44-c6b7ed4a5410)